CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [9.0.2]
+
+### Changed
+- The `SRG_GSLC` job type can now use multiple GPU instance types from the G5, G6, and G6e instance families.
+
```

Comment:
Something like this to account for the fact that both `SRG_GSLC` and `SRG_TIME_SERIES` are affected?
```suggestion
- The `SrgGslc` custom compute environment now includes multiple GPU instance types from the G5, G6, and G6e instance families.
```